src/components/ScreenWrapper.js

```diff
@@ -34,6 +37,13 @@ const propTypes = {
         // Method to attach listener to Navigation state.
         addListener: PropTypes.func.isRequired,
     }),
+
+    /** Details about any modals being used */
+    modal: PropTypes.shape({
+        /** Indicates if there is a modal currently visible or not */
```

Comment:
NAB: Your property names are pretty self-describing, which is great, so the doc comment is unnecessary.